library/src/main/java/com/sothree/slidinguppanel/SlidingUpPanelLayout.kt

```diff
@@ -1184,75 +1298,11 @@ class SlidingUpPanelLayout @JvmOverloads constructor(context: Context, attrs: At
             val collapsedTop = computePanelTopPosition(0f)
             val expandedTop = computePanelTopPosition(maxSlideOffset)
             return if (isSlidingUp) {
-                Math.min(Math.max(top, expandedTop), collapsedTop)
+                min(max(top, expandedTop), collapsedTop)
             } else {
-                Math.min(Math.max(top, collapsedTop), expandedTop)
+                min(max(top, collapsedTop), expandedTop)
             }
         }
     }
 
-    companion object {
```

Comment:
Please add companion object again

Reply:
can you explain why? The companion is unnecessary here and adds an instance creation, wasting resources

Reply:
This is a point